testing/tests/DevExpress.ui.widgets.scheduler/integration.recurringAppointments.tests.js

```diff
@@ -1472,3 +1472,36 @@ QUnit.test("Appointment has correct occurrences dates with interval > 1, custom
     assert.roughEqual(fourthPosition.top - firstPosition.top, eighthPosition.top - fourthPosition.top, 0.5, "Appointment's top are correct");
     assert.roughEqual(seventhPosition.top, eighthPosition.top, 0.5, "Appointment's occurrences after WKST are positioned correct on top");
 });
+
+QUnit.test("Recurrence appointment occurrences should have correct start date with timezone changing (T818393)", function(assert) {
+    this.createInstance({
+        views: ["day", "week", "workWeek", "month"],
+        currentView: "week",
+        startDayHour: 1,
+        firstDayOfWeek: 2,
+        height: 600,
+        dataSource: [{
+            text: "Recurrence",
+            startDate: new Date(2019, 2, 30, 2, 0),
+            startDateTimeZone: 'Europe/Copenhagen',
+            endDate: new Date(2019, 2, 30, 10, 0),
+            endDateTimeZone: 'Europe/Copenhagen',
+            recurrenceException: "",
+            recurrenceRule: "FREQ=DAILY"
+        }],
+        timeZone: 'Europe/Copenhagen',
+        currentDate: new Date(2019, 2, 30)
+    });
+
+    assert.equal(this.scheduler.appointments.getAppointmentCount(), 3, "Appointment has right count of occurrences");
+
+    var $firstAppointment = $(this.scheduler.appointments.getAppointment(0)),
+        firstAppointmentCoords = translator.locate($firstAppointment),
+        $thirdAppointment = $(this.scheduler.appointments.getAppointment(2)),
+        thirdAppointmentCoords = translator.locate($thirdAppointment);
+
+    assert.equal(firstAppointmentCoords.top, thirdAppointmentCoords.top, "Appointment first and third occurrences have same top coordinate");
+
+    assert.equal($firstAppointment.find(".dx-scheduler-appointment-content-date").eq(0).text(), $thirdAppointment.find(".dx-scheduler-appointment-content-date").eq(0).text(), "Appointment first and third occurrences have same start date text");
+    assert.equal($firstAppointment.find(".dx-scheduler-appointment-content-date").eq(2).text(), $thirdAppointment.find(".dx-scheduler-appointment-content-date").eq(2).text(), "Appointment first and third occurrences have same end date text");
+});
```

Comment:
The last two tests are not necessary. They check the text, which was originally correct and there is no code for this.